src/components/molecules/card/card.html

```diff
@@ -0,0 +1,9 @@
+<div class="card-body">
+  <h2 class="card-title">CONCEPT</h2>
+  <img src="../../../img/card-1.jpg" class="card-image">
```

Comment:
```suggestion
<div class="img-wrapper">
  <img src="../../../img/card-1.jpg" class="card-image">
</div>
```

画像を囲う要素を追加して、それに対して `overflow-hidden` を設定するとカードからはみ出なくなるかと思います！


```scss
.img-wrapper {
  overflow: hidden;
  
  .card-image: {
    // いろいろ
  }
}
```